cantabular/queries.go

```diff
@@ -600,6 +600,16 @@ query ($dataset: String!, $variables: [String!]! ) {
 	}
 }`
 
+const QueryValues = `
+query($dataset: String!, $variables: [String!]!, $filters: [Filter!]) {
+	dataset(name: $dataset) {
+		table(variables: $variables, filters: $filters) {
+			values
+            error
+		}
+	}
+}`
```

Comment:
Indentation doesn't look right to me here

Reply:
This has now been fixed.